### DISCUSSION TITLE
Expected return in black litterman

### DISCUSSION THREAD

**Discussion by FeCol150** [dcajasn/Riskfolio-Lib#188] [General]
Hi Dany,

I am actually working on black Litterman to the detail and riskfolio is a greaat tool to do so. However, when I run BL and I extract the expected vector return. It is not clear for me, what the model does and sometimes it is not similar to the past returns, either to the expected view I upload.

Could you please confirm what the model does?

**Comment by dcajasn** [comment]
Hi @FeCol150,

Here is the link https://riskfolio-lib.readthedocs.io/en/latest/parameters.html#ParamsEstimation.black_litterman of the method used.
The black litterman parameter is saved in port.mu_bl and port.cov_bl.

Best regards,
Dany
